Ladybird/CMakeLists.txt

```diff
@@ -81,6 +81,8 @@ if (ENABLE_QT)
     set(CMAKE_AUTORCC ON)
     set(CMAKE_AUTOUIC ON)
     find_package(Qt6 REQUIRED COMPONENTS Core Widgets Network Multimedia)
+elseif (APPLE)
```

Comment:
Might be worth inventing a serenity_dependent_option wrapper thingy for choosing the correct backend, but definitely a "later" idea.